LICENSE.txt

```diff
@@ -1,7 +1,7 @@
 Mule ESB is distributed under the terms of the Common Public Attribution 
 License (CPAL), whose definition follows below.  For information on the license
 type for each third-party software product used by Mule, refer to 
-http://www.mulesoft.org/documentation/display/MULE3INTRO/Third-party+Software+in+Mule 
+https://developer.mulesoft.com/docs/display/current/Third-Party+Software+In+Mule
```

Comment:
to only change this file accomplishes nothing. You also have to run mvn license:format on the project root so that this header is updated on all the source files.